src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/PublicLicenceServiceIntegrationTest.kt

```diff
@@ -0,0 +1,66 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.integration
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.test.context.jdbc.Sql
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.repository.LicenceRepository
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.licence.LicenceStatus
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.licence.LicenceSummary
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.licence.LicenceType
+
+class PublicLicenceServiceIntegrationTest : IntegrationTestBase() {
+
+  @Autowired
+  lateinit var licenceRepository: LicenceRepository
+
+  @Test
+  @Sql(
+    "classpath:test_data/seed-licence-id-1.sql",
+  )
+  fun `Get licences by CRN`() {
+    val resultList = webTestClient.get()
+      .uri("/public/licence-summaries/crn/CRN1")
+      .accept(MediaType.APPLICATION_JSON)
+      .headers(setAuthorisation(roles = listOf("ROLE_VIEW_LICENCES")))
+      .exchange()
+      .expectStatus().isOk
+      .expectHeader().contentType(MediaType.APPLICATION_JSON)
+      .expectBodyList(LicenceSummary::class.java)
+      .returnResult().responseBody
+
+    assertThat(resultList?.size).isEqualTo(1)
+
+    val result = resultList?.first()
+
+    assertThat(result?.id).isEqualTo(1L)
```

Comment:
it'd be kind of nice to check date format.. some other projects make more use of jsonpath rather than deserializing the response..

Reply:
maybe not worth doing now but worth considering in the future

Reply:
Done